components/CancellationList.vue

```diff
@@ -237,4 +237,10 @@ export default {
 #waiting-lists-table {
   margin-bottom: 20px;
 }
+
+::v-deep .clinics-table {
+  table > thead > tr > th {
+    font-size: 1rem;
```

Comment:
This is just for showing it works. Maybe 0.875rem (equivalent to 14px) would be a better choice

Reply:
Updated to 0.875rem as it looks less aggressive. Screenshot and PR body updated accordingly